docs/topics/proc_installing-launcher-operator.adoc

```diff
@@ -92,9 +103,23 @@ kind: Launcher
 metadata:
   name: launcher
 spec:
+
 # OpenShift Configuration
   openshift:
     consoleUrl: # Replace this value with the URL you use to access your OpenShift Web Console (for example: 'https://192.168.64.4:8443').#
+
+# OAuth Configuration
+  oauth:
+    enabled: true
+    keycloakUrl: http://keycloakUrl # TODO: What is the approriate URL to use instead of the default here?#
```

Comment:
@gastaldi What should `http://keycloakUrl` correspond to here? where can the user obtain it?
(or is the value actually supposed to be `http://keycloakURL`? )

Reply:
There isn't a default value. That URL depends on where the Keycloak server is installed

Reply:
It should not be provided if you're not using Keycloak

Reply:
Thank you, I added a comment indicating that it is a replaceable values, and another comment indicating that the Keycloak-related data should be omitted if the user is not using a Keycloak server for authorization. Please let me know what you think.